app/export.go

```diff
@@ -120,19 +148,146 @@ func (a *App) BulkExport(ctx request.CTX, writer io.Writer, outPath string, opts
 	}
 
 	if opts.IncludeAttachments {
-		ctx.Logger().Info("Bulk export: exporting file attachments")
+		mlog.Info("Bulk export: exporting file attachments")
+		exportFilename := a.GetId() + "_export.log"
 		for _, attachment := range attachments {
+			isExist, err := a.FileExists(*attachment.Path)
+			if err != nil {
+				return err
+			}
+			if !isExist {
+				f, err := os.OpenFile(filepath.Join("./data/export/", exportFilename), os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0600)
+				if err != nil {
+					return model.NewAppError("openFile", "Failed to open file", nil, err.Error(), http.StatusInternalServerError)
+				}
+				defer f.Close()
+				ctime, err := strconv.ParseInt(strconv.FormatInt(attachment.CreateAt, 10), 10, 64)
+				if err != nil {
+					return model.NewAppError("parseTime", "Failed to parse create at time", nil, err.Error(), http.StatusInternalServerError)
+				}
+				utime, err := strconv.ParseInt(strconv.FormatInt(attachment.UpdateAt, 10), 10, 64)
+				if err != nil {
+					return model.NewAppError("parseTime", "Failed to parse update at time", nil, err.Error(), http.StatusInternalServerError)
+				}
+				create_time := time.UnixMilli(ctime)
```

Comment:
we dont use _ in our naming using camel case only

Reply:
Okk